app/components/form/instance-table.hbs

```diff
@@ -1,41 +1,46 @@
-{{#if this.initialized}}
-  <AuToolbar @size="large" class="au-u-padding-bottom-none" as |Group|>
-    <Group>
-      <AuHeading @skin="2">Beheer
-        {{this.formInfo.form.definition.id}}
-        Form Instances</AuHeading>
-    </Group>
-    <Group>
-      <AuButton {{on "click" @onCreate}}>Voeg form instance toe</AuButton>
-    </Group>
-  </AuToolbar>
+<AuToolbar @size="large" class="au-u-padding-bottom-none" as |Group|>
+  <Group>
+    <AuHeading @skin="2">Beheer
+      {{this.formInfo.form.definition.id}}
```

Comment:
Shouldn't this be this.formInfo.formDefinition.id?

Reply:
good catch, seems like it was broken before as well.